src/PageComponents/ContentContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import {
+  Flex,
+  FormControl,
+  Switch,
+  Input,
+  InputGroup,
+  InputLeftAddon,
+  FormLabel,
+  Select,
+  Button,
+  Stack
+} from "@chakra-ui/react";
+
+export const ContentContainer = () => (
```

Comment:
This is great, but could you guys try to componentize different parts of the ContentContainer component? Like frontend/backend switches can be a component, name input can be its own component, and the grade level selection can be its own component.